components/alerts/alerts.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alerts</title>
+    <link rel="stylesheet" href="/CSS/utilities.css">
+    <link rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.6.0/styles/atom-one-dark.min.css"
+        integrity="sha512-Fcqyubi5qOvl+yCwSJ+r7lli+CO1eHXMaugsZrnxuU4DVpLYWXTVoHy55+mCb4VZpMgy7PBhV7IiymC0yu9tkQ=="
+        crossorigin="anonymous" />
+</head>
+
+<body>
+    <header class="nav">
+        <nav class="nav-header">
+            <div class="t2">
+                <a href="/index.html">Keviv UI</a>
+            </div>
+            <div class="nav-components">
+                <ul class="nav-component-list">
+                    <li class="nav-pills"><a href="#" class="active">Alert</a></li>
+                    <li class="nav-pills"><a href="#">Avatar</a></li>
```

Comment:
add links to other pages

Reply:
will add once I start working on other pages, I haven't created those pages yet